code/modules/recycling/sortingmachinery.dm

```diff
@@ -277,18 +279,17 @@
 		for(var/obj/structure/bigDelivery/O in src)
 			deliveryCheck = 1
 			if(O.sortTag == 0)
-				O.sortTag = 1
+				O.sortTag = defaultDestinationTag
 		for(var/obj/item/smallDelivery/O in src)
 			deliveryCheck = 1
 			if(O.sortTag == 0)
-				O.sortTag = 1
+				O.sortTag = defaultDestinationTag
 		if(deliveryCheck == 0)
-			H.destinationTag = 1
+			H.destinationTag = defaultDestinationTag
 
 		sleep(10)
 		playsound(src, 'sound/machines/disposalflush.ogg', 50, 0, 0)
 		sleep(5) // wait for animation to finish
-
 		H.init(src)	// copy the contents of disposer to holder

```

Comment:
_Or simply make the delivery chutes separate mixed tags into separate `deliverychutes`_